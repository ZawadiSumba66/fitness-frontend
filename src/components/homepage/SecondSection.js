import { Link } from '@reach/router';
import second from '../../assets/second.jpg';

const SecondSection = () => (
  <div className=" pt-5 mx-lg-5">
    <h5 className="text-sm-gray text-center my-2 text-orange font-weight-bold text-uppercase">
      Reliable and well-sourced health practises
    </h5>
    <h6 className="text-center mb-3">
      Get access to the tips anywhere, anytime.
    </h6>
    <div className="row align-items-center">
      <div className="col-md-6">
        <div className="pl-3">
          <h2 className="text-3xl md:text-3xl font-bold">
            Change your life today for a better tomorrow
          </h2>
          <p className="text-sm-gray mb-5 mt-3">
            Would you like to see a change in your body?
            Are you ready for a whole new healthy lifestyle just by few tips
            offered by our experts? Then worry no more because we got you.
            All you need is positive attitude and the willingness to start.
          </p>
          <Link to="/signup" className="button-orange text-light font-weight-bold px-4 py-2 mt-4">
            LEARN MORE
          </Link>
        </div>
      </div>
      <div className=" col-md-6">
        <img
          src={second}
          alt="FitOne"
          className="second_image"
        />
      </div>
    </div>
  </div>
);

export default SecondSection;
