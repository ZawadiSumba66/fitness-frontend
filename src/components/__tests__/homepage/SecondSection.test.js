import renderer from 'react-test-renderer';
import SecondSection from '../../homepage/SecondSection';

it('matches the second section snapshot', () => {
  const tree = renderer.create(<SecondSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
