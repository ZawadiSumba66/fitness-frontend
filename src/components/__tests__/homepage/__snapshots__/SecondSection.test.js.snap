// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the second section snapshot 1`] = `
<div
  className=" pt-5 mx-lg-5"
>
  <h5
    className="text-sm-gray text-center my-2 text-orange font-weight-bold text-uppercase"
  >
    Reliable and well-sourced health practises
  </h5>
  <h6
    className="text-center mb-3"
  >
    Get access to the tips anywhere, anytime.
  </h6>
  <div
    className="row align-items-center"
  >
    <div
      className="col-md-6"
    >
      <div
        className="pl-3"
      >
        <h2
          className="text-lg-black font-weight-bold"
        >
          Change your life today for a better tomorrow
        </h2>
        <p
          className="text-sm-gray mb-5 mt-3"
        >
          Would you like to see a change in your body? Are you ready for a whole new healthy lifestyle just by few tips offered by our experts? Then worry no more because we got you. All you need is positive attitude and the willingness to start.
        </p>
        <a
          className="button-orange text-light font-weight-bold px-4 py-2 mt-4"
          href="/signup"
          onClick={[Function]}
        >
          LEARN MORE
        </a>
      </div>
    </div>
    <div
      className=" col-md-6"
    >
      <img
        alt="FitOne"
        className="second_image"
        src="second.jpg"
      />
    </div>
  </div>
</div>
`;
